src/main/kotlin/it/pagopa/ecommerce/payment/requests/services/PaymentRequestsService.kt

```diff
@@ -157,4 +159,9 @@ class PaymentRequestsService(
   }
 
   fun getDueDateString(date: XMLGregorianCalendar?): String? = date?.toString()
+
+  private fun dueDateToLocalDate(date: XMLGregorianCalendar): String =
```

Comment:
Looking at Node VerifyPaymentNoticeResponse [due date is optional](https://services.w3.org/xslt?amp;transform=Submit&xmlfile=https://raw.githubusercontent.com/pagopa/pagopa-infra/main/src/core/api/nodopagamenti_api/nodeForPsp/v1/nodeForPsp.wsdl&xslfile=http://tomi.vanek.sk/xml/wsdl-viewer.xsl#src.d1e407) so it have to be process as an optional values also here
```suggestion
  private fun dueDateToLocalDate(date: XMLGregorianCalendar?): String? =
```

Reply:
Fixed in 34decc6